src/lib/receiver.js

```diff
@@ -49,6 +56,7 @@ function createReceiver (opts) {
   const sspReceiverId = SSP_RECEIVER_ID_PREFIX + receiverId + '.'
   debug('receiver id: ' + receiverId)
 
+  const reviewPayment = opts.reviewPayment || (() => Promise.resolve(null))
```

Comment:
Additionally, most users won't have any backend to determine whether to accept payments, and so they'd end up writing a function like this most of the time anyways.